CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ The format is based on [Keep a Changelog], and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- Added the content of 'github.com/dogmatiq/envelopespec' repository into
+  `envelope` subdirectory.
```

Comment:
```suggestion
- Add `envelopespec` package
```

I don't think there's any value to referring to this old repo which will be removed.